### ISSUE TITLE
ePub and PDF document node icons are the same

### ISSUE THREAD

**Issue by ivanistheone** [learningequality/kolibri#4564]
### Observed behavior

…

### Expected behavior

…

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

…

### Context

…

**Comment by indirectlylit** [comment]
@khangmach want to find us a new icon and color?

**Comment by khangmach** [comment]
Since lessons already uses a book icon, for EPUBS, we could use something like: 
https://material.io/tools/icons/?search=chrome&icon=chrome_reader_mode&style=baseline

RE: color
I was thinking about color bloat a while back and was thinking one way we can sort of combat this is to assign colors to various "types" of engagement. Some examples of types of engagement...
One color for Passive engagement -> reading/watching a doc/epub/video
One color for Active engagement -> doing an exercise/interactive html game

We already do this with "inactive" types of engagement by coloring "things you navigate through", (e.g. topics and channel cards) ta dark grey color. 

Check out an example with drive
<img width="560" alt="screen shot 2018-12-12 at 10 51 18 am" src="https://user-images.githubusercontent.com/25045166/49892136-16732c00-fdfd-11e8-9dc9-ac851734f39a.png">

**Comment by rtibbles** [comment]
We base icons on the `kind` at the moment, and both PDFs and EPUBS are 'document' kinds. If the main purpose is to disambiguate the same document presented in two different format,s perhaps they should not be separate content nodes, but the same content node with two different file formats?

**Comment by rtibbles** [comment]
(in which case we would need a better user flow to support two different file formats being attached to the same content node).

**Comment by ivanistheone** [comment]
>  perhaps they should not be separate content nodes, but the same content node with two different file formats?

Sounds like more trouble—as there is no precedent for this (multiply `primary=True` files associated with the same content node) either on Studio or Kolibri.

@rtibbles  How hard would it be to update the choosing-the-icon logic to use kind and file format vs. just kind?

**Comment by rtibbles** [comment]
We mostly insulate the frontend from any consideration of file extensions, the point of content kinds is to have a limited set of 'user interpretable' kinds. Whereas file extensions are a more technical consideration.

What might be better is to have `epub` be a `book` type, as opposed to a document type.

**Comment by ivanistheone** [comment]
> What might be better is to have epub be a book type, as opposed to a document type.

Yeah this would make more sense. Currently the whole `epub` and `pdf` both part of `document` was fairly confusing on ricecooker + studio side. I propose we continue discussion in January when Jordan is back, but I'm sure she'll be on board with splitting the `epub` into its own content kind.

Note: doing this switch will require re-running all channels that used `epub` to switch them to the new convention, but I don't think there were are many of them. Not sure if we'll have to do some backward compatible to things already published...

**Comment by indirectlylit** [comment]
>  If the main purpose is to disambiguate the same document presented in two different formats, perhaps they should not be separate content nodes, but the same content node with two different file formats?

I agree with this

> there is no precedent for this (multiple `primary=True` files associated with the same content node)

The curator or chef should choose one as being "better" and make it primary, while the other becomes a secondary alternative format.

> What might be better is to have `epub` be a `book` type, as opposed to a document type.

I think we might eventually need to decouple our implementation details a bit from the user experience:

* Are the African Storybooks no longer 'books' because it's a HTML app?
* Are OER math textbooks no longer 'books' because it's in PDF?
* Is a powerpoint presentation really a 'document' just because it's a PDF?

Some higher-level questions

* What are we actually trying to communicate to users when we label content as a certain kind?
* When are users interested in the render format and learner UX, and when are they interested in the content itself?
* Would we ever allow curators to re-label the kind of content nodes?
